js/source/scripts.js

```diff
@@ -11,101 +11,113 @@
 // - https://drupal.org/node/1446420
 // - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
 
-(function (Drupal, $, window) {
```

Comment:
Looks like you undid the changes I pushed to the other branch. Please restore these.